### PR TITLE
Implement a solution template

### PR DESCRIPTION
**Pull request by nekotuko**

### CODE REVIEW

File: .vscode/problem-solving.code-workspace [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+	"folders": [
+		{
+			"path": ".."
+		}
+	],
+	"settings": {
```
Comment (suad-bakhshiyev) [inline .vscode/problem-solving.code-workspace:7]: This is great to keep the workspace clean! Lmk if you agree!

**Comment by suad-bakhshiyev** [comment]
Should we merge it today?

**Review by suad-bakhshiyev — CHANGES_REQUESTED** [review]

File: .vscode/problem-solving.code-workspace (hunk 1 above)
Comment (nekotuko) [reply .vscode/problem-solving.code-workspace:7]: I agree!

File: buildSrc/src/main/kotlin/java-solution.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+java { toolchain { languageVersion.set(JavaLanguageVersion.of(17)) } }
+
+dependencies {
+    // Use JUnit Jupiter for testing.
+    testImplementation("org.junit.jupiter:junit-jupiter:5.9.1")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+
+    // This dependency is used by the application.
+    implementation("com.google.guava:guava:31.1-jre")
+}
```
Comment (suad-bakhshiyev) [inline buildSrc/src/main/kotlin/java-solution.gradle.kts:15]: Looks great!

Comment (nekotuko) [reply buildSrc/src/main/kotlin/java-solution.gradle.kts:15]: Thanks!

**Review by suad-bakhshiyev — APPROVED** [review]

**Pull request merged**